setup/upgrade_major.rst

```diff
@@ -120,6 +127,48 @@ done!
 
         $ SYMFONY_DEPRECATIONS_HELPER=max[total]=999999 php ./bin/phpunit
 
+Deprecations in logs
+~~~~~~~~~~~~~~~~~~~~
+If you use the Symfony default configuration, a part of the ``monolog`` configuration for the prod environment can be uncommented allowing to log deprecations in a ``prod.deprecations.log`` file:
+
+.. code-block:: yaml
+
+    monolog:
+        handlers:
+            # ...
+            deprecation:
+                type: stream
+                path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
+            deprecation_filter:
+                type: filter
+                handler: deprecation
+                max_level: info
+                channels: ["php"]
+
+Under the hood, deprecations are logged on the ``php`` Monolog channel at ``INFO`` psr-3 level. But as soon as an extra ``deprecation`` channel exists; deprecations are logged into this channel instead. So you can log deprecations on this dedicated channel and make the ``deprecation_filter`` useless:
+
+.. code-block:: yaml
+
+    monolog:
+        channels:
+            - deprecation
+        handlers:
+            # ...
+            deprecation:
+                type: stream
+                channels: ["deprecation"]
+                path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
+
+.. versionadded:: 5.1
+
+    The deprecation logging into a dedicated channel was introduced in Symfony 5.1.
+
+Road to Code Deprecation Free
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Track the deprecations from the PHPUnit bridge and/or (depending on your test coverage) browse your application and note the deprecations on the web debug toolbar.
+2. Deploy your code (that should not have deprecations anymore) on production and check for a while that no remaining deprecations are logged into ``prod.deprecations.log``.
```

Comment:
I would write instead

And check from time to time the content of this log file to be aware of the remaining deprecations you have to change/update

Wdyt?
PS i use it for a 5.1 it is very usefull :)

Reply:
Updated this way :)